### PR TITLE
TC_03.04.004 | Create Pipeline project4

### PR DESCRIPTION
**Pull request by AlYeMax**
https://trello.com/c/SMSeoNJU/105-tc0304004-create-pipeline-project

### CODE REVIEW

File: cypress/e2e/createPipelineProject4.cy.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/// <reference types="cypress"/>
+
+describe('createPipelineProject', () => {
+
+  it('TC_03.04.004 | Create Pipeline project4', () => {
+      cy.get("a[href='/view/all/newJob']").click();
+      cy.get('#name').type(`Pipeline project4`);
```
Comment (SvetlanaMag) [inline cypress/e2e/createPipelineProject4.cy.js:7]: Please have a look at US_03.04 and p.3 'spaces between words are not allowed', so you need to change the name of project.

**Comment by SvetlanaMag** [comment]
Please use fixtures for pipeline name.

**Comment by AlYeMax** [comment]
OK!

**Review by SvetlanaMag — APPROVED** [review]

**Comment by Anastassyia** [comment]
Looks very good!

**Review by Anastassyia — APPROVED** [review]

**Pull request merged**